src/VssDatabase_Record.cpp

```diff
@@ -0,0 +1,98 @@
+#include "VssDatabase_Record.hpp"
+
+#include <string>
+#include <memory>
+#include <iostream>
+
+VssDatabase_Record::VssDatabase_Record(std::shared_ptr<ILogger> loggerUtil, std::shared_ptr<ISubscriptionHandler> subHandle, std::string recordPath)
+:overClass_(loggerUtil,subHandle)
+{
+    
+    std::string workingDir = recordPath;
+
+    if(workingDir.empty())
+    {
+        workingDir = getexepath();
+        workingDir.replace(workingDir.find("kuksa-val-server"),workingDir.size()-workingDir.find("kuksa-val-server"),"");
+        workingDir += "logs";
+    }
+
+    logfile_name_ = "record-%Y%m%d_%H%M%S.log.csv";
```

Comment:
```suggestion
    logfile_name_ = "record-%Y%m%d.log.csv";
```
to avoid number of log files?